### DISCUSSION TITLE
Multi-account: Where to find the profile name?

### DISCUSSION THREAD

**Discussion by erasariel** [rfvgyhn/min-ed-launcher#75] [Q&A] [answered]
Hi - many thanks for making this launcher possible. I'm not very savvy so I've been having a hard time figuring out the multi-account feature. I have a Steam-linked account and another Frontier account that I currently just log in through the normal launcher. How do I find the profile-name value needed by the /frontier argument?

Any help would be much appreciated, thank you!

**Comment by rfvgyhn** [comment, marked as answer]
It's min-ed-launcher specific, so you can make up whatever name you'd like.

**Reply by erasariel** [reply]
Thanks for the prompt answer! So just to triple check, is this the right way of getting multi-account to work on Steam with a second non-connected Frontier account?

1. Log into the Elite launcher with commander 1
2. Launch min-ed-launcher with argument /frontier cmdr1, then close ED
3. Log out of the Elite launcher and login with commander 2
4. Launch min-ed-launcher with argument /frontier cmdr2, then close ED
5. From that point on, all I need to do is launch min-ed-launcher with either /frontier cmdr1 or /frontier cmdr2 to choose the profile I like

**Reply by rfvgyhn** [reply]
* Unless it's a brand new account that hasn't been setup with a frontier account, you don't need to do anything with the default launcher
* /frontier usage will only be with the frontier account. The steam account won't use the /frontier flag at all

If it were me, I'd create shortcuts for each:
1. In steam, set Elite's launch options to what I wanted for both accounts. `cmd /c "MinEdLauncher.exe %command% /autorun /autoquit /edh4"`
2. Create two shortcuts that launch elite
    1. right click my desktop
    2. New > Shortcut
    3. Select the steam executable as the item (i.e. C:\Program Files (x86)\Steam\Steam.exe)
    4. Modify each shortcut to tell steam to launch Elite by appending the following to the _target_ field. `-gameidlaunch 359320`
3. Modify one of the shortcuts to target my second account by appending the /frontier flag
    1. Cmdr1 Shortcut:
        ![cmdr1](https://user-images.githubusercontent.com/731861/205177672-054bd078-2892-4871-80ef-c90efd0fafd7.png)
    2. Cmdr2 Shortcut:
        ![cmdr2](https://user-images.githubusercontent.com/731861/205179154-78656aad-af98-4752-9cc3-a02fe4d792c7.png)
5. Use the shortcuts to launch the desired profile

**Reply by erasariel** [reply]
That definitely clears things up, I fully understand now. Thank you so much for taking the time to explain it to a less savvy person! Marking as answered.
